frontend/components/droplets/lessons/lesson-renderer.tsx

```
@@ -451,61 +450,106 @@ function convertSingleBlock(blockAny: any, blockIndex: number): Block | null {
-          .join("");
-
-        return rowIndex === 0
-          ? `<thead><tr>${cells}</tr></thead>`
-          : `<tr>${cells}</tr>`;
+      const hasHeaders = blockAny.props?.headers !== false; // Default to true if not specified
+      const cellBackgroundColors: Record<string, string> = {};
+
+      // Convert table to markdown (GFM format)
+      const markdownRows: string[] = [];
+
+      tableContent.rows.forEach((row: any, rowIndex: number) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)